src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyActions.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import {t} from 'app/locale';
+
+type Props = {
+  onCancel: () => void;
+  onSave: () => void;
+};
+
+const ProjectDataPrivacyActions = ({onCancel, onSave}: Props) => (
+  <Wrapper>
+    <StyledButton onClick={onCancel}>{t('Cancel')}</StyledButton>
+    <StyledButton priority="primary" onClick={onSave}>
+      {t('Save Rules')}
+    </StyledButton>
+  </Wrapper>
+);
+
+export default ProjectDataPrivacyActions;
+
+const Wrapper = styled('div')`
+  display: flex;
+  justify-content: flex-end;
+  > * {
+    margin-right: 14px;
+    :last-child {
+      margin-right: 0;
+    }
+  }
+`;
+
+const StyledButton = styled(Button)`
+  width: 124px;
```

Comment:
Can we utilize `size` prop of the Button component instead of setting plain number width?

Reply:
I thought about using the `size` prop, but it  doesn't behave as I would like

Reply:
I would stick to the closest standard button size we have.